src/lib/commandBarConfigs/settingsCommandConfig.ts

```diff
@@ -129,10 +125,16 @@ export function createSettingsCommand({
     icon: 'settings',
     needsReview: false,
     onSubmit: (data) => {
-      if (data !== undefined && data !== null) {
-        send({ type: `set.${type}`, data })
+      if (
+        data !== undefined &&
+        data !== null &&
```

Comment:
These are probably redundant, no? If the two below conditions are true, surely, these cannot be?